grails-app/controllers/org/pih/warehouse/api/ProductSupplierApiController.groovy

```diff
@@ -28,4 +30,17 @@ class ProductSupplierApiController {
         productSupplierService.delete(params.id)
         render status: 204
     }
+
+    def create(ProductSupplierDetailsCommand productSupplierCommand) {
+        ProductSupplier productSupplier = productSupplierService.saveDetails(productSupplierCommand)
```

Comment:
I wonder if the word `create` would not be more suitable here instead of `save`, but I'll let @jmiranda decide what he prefers here. Other than that and one minor naming thing above it LGTM.

Reply:
to include my opinion - I'd stick with `save`, since we are actually calling the db and saving the entity (and GORM's method is named `save`, not `create`, so to kinda follow the convention)
But I don't mind both, so let me know what you guys prefer.
I can also see that Grails' docs probably follow `save`/`update` pattern: https://docs.grails.org/latest/guide/REST.html

Reply:
Looking into this docs, it also suggests calling the action `save` if you do POST to the `/productSuppliers`, so then if you'd like to follow it, then you need to change the action name to `save`.

Reply:
Sounds good.